src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormatTest.java

```diff
@@ -0,0 +1,34 @@
+package io.cdap.plugin.gcp.bigquery.sink;
+
+import org.apache.hadoop.mapreduce.JobContext;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
```

Comment:
Test is not actually testing the change

Reply:
Added some mocked unit tests to actually verify invocations.